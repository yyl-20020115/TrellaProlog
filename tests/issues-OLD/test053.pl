:-initialization(main).

main :-
	X1 =.. [1],
	writeq(X1), nl,
	X2 =.. [1,2,3],
	writeq(X2), nl.
