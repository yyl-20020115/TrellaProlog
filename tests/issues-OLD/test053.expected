1
   error(type_error(atom,1),(=..)/2).
